### PR TITLE
do not activate DASD devices after formatting (bsc#1187012)

### PR DESCRIPTION
**Pull request by wfeldt**
## Task

Port https://github.com/yast/yast-s390/pull/89 to SLE15-SP2.

## Original task

- https://bugzilla.suse.com/show_bug.cgi?id=1187012
- https://trello.com/c/N39SyWPy

After formatting DASD devices, `yast dasd` shows `Error: Cannot set use_diag='0' while online='1'`.

The cause is that `dasd_configure` has a bit of a problem handling the `use_diag` setting. See [this analysis](https://bugzilla.suse.com/show_bug.cgi?id=1187012#c45) for details.

## Solution

The bottom line is not to try to activate an already active DASD with `dasd_configure`. And since they have just been formatted, they are obviously already active. So simply skip this step.

### CODE REVIEW

File: package/yast2-s390.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jun 18 12:05:42 UTC 2021 - Steffen Winterfeldt <snwint@suse.com>
+
+- do not activate DASD devices after formatting (bsc#1187012)
+- 4.2.6
```
Comment (jreidinger) [inline package/yast2-s390.changes:5]: I miss spec file change

**Review by jreidinger — APPROVED** [review]
